x-pack/plugins/canvas/public/components/sidebar_header/__examples__/custom_element_modal.examples.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { CustomElementModal } from '../custom_element_modal';
+// @ts-ignore
+import { elasticLogo } from '../../../lib/elastic_logo';
```

Comment:
If you put `.png` at the end, TS will know what this is.

Reply:
`elasticLogo` is actually an exported JS variable with the base 64 data URL string of the elastic logo image. I converted the file to TS to resolve the error and removed the `@ts-ignore`.